mediaQueries.css

```diff
@@ -0,0 +1,324 @@
+@media only screen and (min-width: 768px) {
+  header {
+    position: absolute;
+    width: 100%;
+    height: 44px;
+    left: 1px;
+    top: 1px;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .navbar ul li {
+    display: inline;
+  }
+
+  .text-holder {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 100px;
+    width: 500px;
+  }
+
+  .headline {
+    background-image: url("./img/header.png");
+  }
```

Comment:
While you did address the previous reviewer's requirement, you didn't do so properly. The background image for this section is still not placed correctly. Please review the Figma template and make sure the background positioning matches that of the template design.

Reply:
Is it on the Desktop Version or Mobile Version?
Can you clarify it a bit more may be with an image?